schema/UKRDC/views/vwe_extract_radar.SQL

```diff
@@ -33,5 +33,14 @@ WHERE
         WHERE
             pm.programname IN ('RADAR', 'NURTURE') AND
             pm.totime IS NULL
+    )
+    AND
+     NOT EXISTS (
+        SELECT 1
+        FROM programmembership pm2
+        WHERE
```

Comment:
This needs a join on...

`        INNER JOIN patientrecord pr3 ON pm2.pid = pr3.pid`

Then use the ukrdcid as the criteria.